### PR TITLE
Updated setup documentation on how to access Github packages

### PR DESCRIPTION
**Pull request by pimg**

### CODE REVIEW

File: SETUP.md [hunk 1]
```diff
@@ -1,6 +1,79 @@
...
+Go to https://github.com/settings/tokens and click "Generate new token"
+Write a note for the token, for example "MAVEN_PACKAGES_PAT"
+Select read:packages, delete:packages, and write:packages.
+Click Generate Token
+Copy the token, we will need it soon. If you leave this page without copying the token, you will need to generate a new one
+Note: Treat this token as you would treat any password.
+
+#### Setting up settings.xml
+
+1. Go to your maven directory. Default directory on Windows is C:\Users\USERNAME\.m2 . If you don't have this directory, you may need to install Maven first.
```
Comment (nt-as) [inline SETUP.md:29]: I think we should `create it first`. Installing maven will not create the directory (you have to run it first, which in turn will fail if you have not done this series of steps)

File: SETUP.md [hunk 2]
```diff
@@ -1,6 +1,79 @@
...
+2. Create a file called "settings.xml"
+3. Paste the following into the file:
+```
```
Comment (nt-as) [inline SETUP.md:32]:
```suggestion
```xml
```

File: SETUP.md (hunk 1 above)
Comment (pimg) [reply SETUP.md]: fixed.

File: SETUP.md (hunk 2 above)
Comment (pimg) [reply SETUP.md]: fixed

**Review by nt-as — APPROVED** [review]

**Pull request merged**